tests/compas/datastructures/test_halfedge.py

```diff
@@ -240,3 +251,30 @@ def test_del_edge_attribute_in_view(mesh, edge_key):
     del attrs["foo"]
     with pytest.raises(KeyError):
         attrs["foo"]
+
+
+# ==============================================================================
+# Tests - Loops & Strip
+# ==============================================================================
+
+def test_loops_and_strips(sphere):
```

Comment:
It would be good to have tests for each of the terminal `if` statements. For example, the `meridian` below is a closed loop.  A test for the cases that would use  the `vu_loop` I think would be a good idea.

Reply:
you mean to check the loop for a mesh that is not closed?

Reply:
i mean there are many `if` statements that `return` and the given test doesn't check all these 'leaves'.